great_expectations/core/util.py

```diff
@@ -401,15 +403,12 @@ def convert_to_json_serializable(  # noqa: C901 - complexity 32
         return data.to_json_dict()
 
     # PySpark schema serialization
-    if StructType is not None:
-        if isinstance(data, StructType):
-            return dict(data.jsonValue())
+    if StructType and isinstance(data, StructType):
+        return dict(data.jsonValue())
 
-    else:
-        raise TypeError(
-            f"{str(data)} is of type {type(data).__name__} which cannot be serialized."
-        )
-    return None
```

Comment:
This bug was a big deal! Anything that couldn't be serialized was ending up as `None` instead if raising the `TypeError` on line 409 when `StructType` was imported.